src/components/CreateForum.js

```diff
@@ -0,0 +1,296 @@
+import React, { useRef, useState, useEffect } from "react"
+import { Form, Button, Card, Alert } from "react-bootstrap"
+import { Link, useHistory } from "react-router-dom"
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import { format } from 'date-fns'
+import { useAuth } from "../contexts/AuthContext"
+import Paper from "@material-ui/core/Paper";
+import Typography from "@material-ui/core/Typography";
+import NavigationBar from './NavigationBar';
+import MuiAlert from '@mui/material/Alert';
+import Grid from '@material-ui/core/Grid';
+import FormHelperText from '@material-ui/core/FormHelperText';
+import TextField from '@material-ui/core/TextField';
+import { makeStyles } from '@material-ui/core/styles';
+
+const { REACT_APP_API_ENDPOINT } = process.env;
+
+const CreateForum = () => {
+
+    const { currentUser } = useAuth();
+    const history = useHistory()
+
+    //const email = currentUser.email;
+    const [email, setEmail] = React.useState('');
+    const [userID, setUserID] = React.useState('');
+    const [open, setOpen] = React.useState(false);
+
+
+    React.useEffect(() => {
+
+        if (currentUser == null) {
+            history.push("/login");
+        }
+
+        setEmail(currentUser.email);
+        loadUserEmailSearch(currentUser.email);
+    }, []);
+
+    const loadUserEmailSearch = (email) => {
+        callApiGetUserEmailSearch(email)
+            .then(res => {
+                var parsed = JSON.parse(res.express);
+                //console.log(parsed[0].id);
+                setUserID(parsed[0].id);
+            });
+    }
+
+    const callApiGetUserEmailSearch = async (email) => {
+        const url = `${REACT_APP_API_ENDPOINT}/userSearchByEmail`;
+        console.log(url);
+
+        const response = await fetch(url, {
+            method: "POST",
+            headers: {
+                "Content-Type": "application/json",
+            },
+            body: JSON.stringify({
+                email: email
+            })
+        });
+        const body = await response.json();
+        if (response.status !== 200) throw Error(body.message);
+        return body;
+    }
+
+    const [createdForumsList, setCreatedForumsList] = React.useState([]);
```

Comment:
you shouldn't need this, as a local list (in this use case) is not doing anything

Reply:
remove this and the `setCreatedForumsList()` calls